chart/openfaas/templates/faas-idler-dep.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  app: {{ template "openfaas.name" . }}
+  namespace: {{ .Release.Namespace | quote }}
+spec:
+  replicas: {{ .Values.faasIdler.replicas }}
+  template:
+    metadata:
+      labels:
+        app: faas-idler
+    spec:
+      containers:
+        - name: faas-idler
+          image: {{ .Values.faasIdler.image }}
+          imagePullPolicy: {{ .Values.openfaasImagePullPolicy }}
+          env:
+            - name: gateway_url
+              value: {{ .Values.faasIdler.gatewayUrl }}
+            - name: prometheus_host
+              value: {{ .Values.faasIdler.prometheusHost }}
+            - name: prometheus_port
+              value: {{ .Values.faasIdler.prometheusPort }}
```

Comment:
Don't we have this in the gateway config somewhere?

Reply:
Unfortunately we don't

Reply:
I'm sure the gateway can accept a Prometheus port / host.